lib/pos_mempool.go

```diff
@@ -40,7 +39,7 @@ type MempoolIterator interface {
 // by Fee-Time algorithm. More on the Fee-Time algorithm can be found in the documentation of TransactionRegister.
 type PosMempool struct {
 	sync.RWMutex
-	status PosMempoolStatus
+	status *atomic.Int32
```

Comment:
Why migrate this?

Reply:
IMO, adding this thread-safe status to enable the `IsRunning()` function results in more effort to understand the concurrency of the struct, without a clear payoff. 

My suggestion would be to defer introducing it until we know we want the `IsRunning()` function at some point in the future.

Reply:
Note: just use int.